src/components/FacebookFeed.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+
+export class FacebookFeed extends React.Component<any, any> {
+    constructor(props: any) {
+        super(props)
+        this.state = { width: 0, height: 0 }
+        this.updateWindowDimensions = this.updateWindowDimensions.bind(this)
+    }
+
+    componentDidMount() {
+        this.updateWindowDimensions()
+        window.addEventListener('resize', this.updateWindowDimensions)
+    }
+
+    componentWillUnmount() {
+        window.removeEventListener('resize', this.updateWindowDimensions)
+    }
+
+    updateWindowDimensions() {
+        this.setState({ width: window.innerWidth, height: window.innerHeight })
+    }
+
+    render = (): JSX.Element => {
+        if (this.state.width > 1500) {
```

Comment:
I'd try to avoid code duplication by storing the `data-height` value in a variable, like this!
```suggestion
        let dataHeight: string = ''
        if (this.state.width > 1500) dataHeight = '800'
        return (<div data-height={dataHeight}></div>)
```

Reply:
Good idea!